onprc_ehr/resources/queries/study/StudyDetails_phase_values.query.xml

```diff
@@ -0,0 +1,18 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="StudyDetails_Reference_Data" tableDbType="NOT_IN_DB">
+                <tableTitle>Study Details</tableTitle>
+                <columns>
+                    <column columnName="value">
+                        <isKeyField>true</isKeyField>
+                        <isHidden>false</isHidden>
+                    </column>
+                </columns>
+                <sorts>
```

Comment:
Still illegal!

Reply:
lot of illegal business going on, now removed